### PR TITLE
switch to ubuntu trusty 14.04

### PR DESCRIPTION
**Pull request by Fahad-Alsaidi**
This required for Adding CTL support #2284.

### CODE REVIEW

**Comment by Fahad-Alsaidi** [comment]
I need help to fix this:
```
======================================================================

ERROR: TestImagePath.test_path

----------------------------------------------------------------------

Traceback (most recent call last):

  File "/home/travis/build/python-pillow/Pillow/Tests/test_imagepath.py", line 62, in test_path

    p = ImagePath.Path(tuple(arr.tostring()))

SystemError: new style getargs format but argument is not a tuple
```

File: Tests/test_file_icns.py [hunk 1]
```diff
@@ -4,6 +4,12 @@
 
 import sys
 
+# The the default version of PyPy in Trusty is PyPy 5.4.1.
+# There is a known error in it: TypeError: PyPy does not yet implement the new buffer interface (issue #2163).
+
+SKIP_5_4_1_PYPY = hasattr(sys, 'pypy_version_info') and (
+    sys.pypy_version_info >= (5, 4, 1, 'final', 0))
```
Comment (wiredfool) [inline Tests/test_file_icns.py]: Lets make this a little tighter -- either just 5.4.1 or 5.4.0 < v <5.4.1. 

The default case should be that we have to reaffirm this skip for each pypy that fails, not that we never run these tests again.

**Comment by wiredfool** [comment]
The getargs thing is a real bug in the C layer.

**Comment by wiredfool** [comment]
Ok, I was wrong, that's the same underlying issue. 

I've confirmed that there's a regression in pypy 5.4 and 5.6, where it worked in 5.3. There's a workaround in #2294.

**Comment by wiredfool** [comment]
Closed in favor of #2294

**Comment by hugovk** [comment]
Merged via PR https://github.com/python-pillow/Pillow/pull/2294.

**Pull request closed without merging**